beacon-chain/core/blocks/block_operations_test.go

```diff
@@ -1659,7 +1659,6 @@ func TestProcessDeposit_SkipsDepositWithUncompressedSignature(t *testing.T) {
 	a, _ := blsintern.DecompressG2(bytesutil.ToBytes96(dep[0].Data.Signature))
 	uncompressedSignature := a.SerializeBytes()
 	dep[0].Data.Signature = uncompressedSignature[:]
-	ssz.ToggleCache(false)
```

Comment:
No longer needed, SSZ no longer uses a global cache that we can just toggle on/off